lib/features/steps/request_assertion_steps.rb

```diff
@@ -55,6 +55,16 @@ def assert_received_requests(request_count, list, list_name)
   list.sort_by_sent_at! request_count
 end
 
+# Verify that at least a certain amount of requests have been received
+# This step is only intended for use in stress tests
+#
+# @step_input min_received [Integer] The minimum amount of requests required to pass
+# @step_input request_type [String] The type of request (error, session, build, etc)
+Then('I have received at least {int} {word} requests') do |min_received, request_type|
```

Comment:
I think this should be "I have received at least 2 errors" for consistency with "I wait to receive 2 errors" and "I should receive no errors"

```suggestion
Then('I have received at least {int} {word}') do |min_received, request_type|
```